akka-docs/src/main/paradox/stream/operators/Flow/asFlowWithContext.md

```diff
@@ -0,0 +1,19 @@
+# Flow.asFlowWithContext
+
+Turns a Flow into a FlowWithContext which can propagate a context per element along a stream.
+
+@ref[Simple operators](../index.md#simple-operators)
+
+@@@div { .group-scala }
+
+## Signature
+
+@@signature [Flow.scala](/akka-stream/src/main/scala/akka/stream/scaladsl/Flow.scala) { #asFlowWithContext }
+
+@@@
+
+## Description
+
+Turns a Flow into a FlowWithContext which can propagate a context per element along a stream.
+The first function passed into asFlowWithContext must turn incoming elements and contexts into elements of the flow, for every incoming elements.
```

Comment:
Maybe rephrase "must turn each incoming pair of element and context value into an element of this Flow"